include/aws/io/file_utils.h

```diff
@@ -61,6 +61,12 @@ char aws_get_platform_directory_separator(void);
 AWS_IO_API
 struct aws_string *aws_get_home_directory(struct aws_allocator *allocator);
 
+/**
+ * Returns true if a file or path exists, otherwise, false.
+ */
+AWS_IO_API
+bool aws_does_path_exist(const char *path);
```

Comment:
`aws_path_exists()`?
would work well with any future `aws_path_XYZ()` functions we might add

Reply:
yep